packages/components/src/sortTabs/SortTab.js

```diff
@@ -76,7 +81,7 @@ class SortTab extends React.Component<Props> {
             <View
               style={[
                 styles.priceDurationContainer,
-                isMobile && styles.priceTagMobile,
+                mobileLayout && styles.priceTagMobile,
```

Comment:
Can we unify naming of this container? We are currently using `priceDuration` and `priceTag` for the same component.

Reply:
Done!